### ISSUE TITLE
Misleading error message on upgrade with old schema

### ISSUE THREAD

**Issue by carli2** [arangodb/arangodb#6876]
## My Environment

* __ArangoDB Version__:        upgraded from 3.3 to 3.4
* __Deployment Mode__:       Single Server
* __Deployment Strategy__:   systemctl
* __Operating System__:        Fedora
* __Used Package__:              .rpm

__Problem__:
```
2018-10-14T10:01:47Z [15360] ERROR {startup} Database directory version (30316) is lower than current version (30400).
2018-10-14T10:01:47Z [15360] ERROR {startup} ----------------------------------------------------------------------
2018-10-14T10:01:47Z [15360] ERROR {startup} It seems like you have upgraded the ArangoDB binary.
2018-10-14T10:01:47Z [15360] ERROR {startup} If this is what you wanted to do, please restart with the'
2018-10-14T10:01:47Z [15360] ERROR {startup}   --database.auto-upgrade true'
2018-10-14T10:01:47Z [15360] ERROR {startup} option to upgrade the data in the database directory.'
2018-10-14T10:01:47Z [15360] ERROR {startup} Normally you can use the control script to upgrade your database'
2018-10-14T10:01:47Z [15360] ERROR {startup}   /etc/init.d/arangodb stop'
2018-10-14T10:01:47Z [15360] ERROR {startup}   /etc/init.d/arangodb upgrade'
2018-10-14T10:01:47Z [15360] ERROR {startup}   /etc/init.d/arangodb start'
2018-10-14T10:01:47Z [15360] ERROR {startup} ----------------------------------------------------------------------'
```

/etc/init.d/arangodb does not exist; calling "update" with systemctl also fails (unknown operation)

__Expected result__:

improve the error message and tell the correct location of stop/upgrade/start script

**Comment by maxkernbach** [comment]
Hi @carli2,

thanks for reporting this issue. There are known upgrade issues for the latest available 3.4 Release Candidate RC2, which will be addressed in the upcoming version 3.4.0-RC3. 

We will inform you once 3.4.0-RC3, which includes a fix for the upgrade issue, is released.

**Comment by vinaypyati** [comment]
Hi @carli2,

3.4.0-RC3 has been released and it includes a fix for this upgrade issue. You can download it from the [download section](https://www.arangodb.com/download-technical-preview/). 

I am closing this ticket now. Please feel free to comment if the issue persists

Thanks!
